.pre-commit-config.yaml

```diff
@@ -1,51 +1,55 @@
----
 repos:
-    - repo: https://github.com/pre-commit/pre-commit-hooks
-      rev: v4.4.0
-      hooks:
-          - id: trailing-whitespace
-            exclude: __snapshots__
-          - id: end-of-file-fixer
-          - id: check-yaml
-          - id: check-added-large-files
-            exclude: ^(docs/assets/.*|assets/.*)$
-
-    - repo: https://github.com/asottile/reorder-python-imports
-      rev: v3.10.0
-      hooks:
-          - id: reorder-python-imports
-
-    - repo: https://github.com/psf/black
-      rev: 23.7.0
-      hooks:
-          - id: black
-
-    - repo: https://github.com/PyCQA/pylint
-      rev: v3.0.0a6
-      hooks:
-          - id: pylint
-
-    - repo: https://github.com/RobertCraigie/pyright-python
-      rev: v1.1.318
-      exclude: ^typings/
-      hooks:
-          - id: pyright
-
-    - repo: https://github.com/igorshubovych/markdownlint-cli
-      rev: v0.35.0
-      hooks:
-          - id: markdownlint
-          - id: markdownlint-fix
-
-    - repo: https://github.com/jumanjihouse/pre-commit-hook-yamlfmt
-      rev: 0.2.3
-      hooks:
-          - id: yamlfmt
-
-    - repo: https://github.com/dhruvmanila/remove-print-statements
-      rev: v0.5.2
-      hooks:
-          - id: remove-print-statements
-            args:
-                - --verbose
-                - --ignore=tests/conftest.py
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: trailing-whitespace
+        exclude: __snapshots__
+      - id: end-of-file-fixer
+      - id: check-yaml
+      - id: check-added-large-files
+        exclude: ^(docs/assets/.*|assets/.*)$
+
+  - repo: https://github.com/asottile/reorder-python-imports
+    rev: v3.10.0
+    hooks:
+      - id: reorder-python-imports
+
+  - repo: https://github.com/psf/black
+    rev: 23.7.0
+    hooks:
+      - id: black
+
+  - repo: https://github.com/PyCQA/pylint
+    rev: v3.0.0a6
+    hooks:
+      - id: pylint
+
+  - repo: https://github.com/RobertCraigie/pyright-python
+    rev: v1.1.318
+    exclude: ^typings/
+    hooks:
+      - id: pyright
+
+  - repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.35.0
+    hooks:
+      - id: markdownlint
+      - id: markdownlint-fix
+
+  - repo: https://github.com/jumanjihouse/pre-commit-hook-yamlfmt
+    rev: 0.2.3
+    hooks:
+      - id: yamlfmt
+
+  - repo: https://github.com/dhruvmanila/remove-print-statements
+    rev: v0.5.2
+    hooks:
+      - id: remove-print-statements
+        args:
+          - --verbose
+          - --ignore=tests/conftest.py
+
+  - repo: https://github.com/adrienverge/yamllint
```

Comment:
Keep in mind we already have this hook:  https://github.com/jumanjihouse/pre-commit-hook-yamlfmt

could it cause a problem? I mean, will the 2 diffferent yaml linters/formatters interfere with each other? If both are actually needed, then could it be done in such a way that the second one will *only* care about the line length (by ignoring all other rules/formatting?)

Reply:
yes u r right, I have configured one of the linter to ignore line length, by making `yamlfmt` to only check line length